docs/DEVELOPING.md

```diff
@@ -737,6 +737,121 @@ $ make ftest NAME=mssql
 
 ℹ️ The e2e test uses default values defined in [Configure Microsoft SQL connector](#configure-microsoft-sql-connector)
 
+## Confluence Connector
+
+The [Elastic Confluence connector](../connectors/sources/confluence.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.8_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+Confluence versions 7 or later are compatible with Elastic connector frameworks.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Confluence details](#gather-confluence-details)
+2. [Configure Confluence connector](#configure-confluence-connector)
+
+#### Gather Confluence details
+
+Collect the information that is required to connect to your Confluence instance:
+
+- The server host url where Confluence is hosted.
+- Username for the Confluence server or service account for the Confluence cloud.
+- Password for the Confluence server or API token for the Confluence cloud.
+- SSL certificate for a secure connection
+
+#### Configure Confluence connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `is_cloud`
+
+Flag to determine the Confluence platform type. `True` if Confluence cloud and `False` if Confluence server. Default value is `False`.
+
+##### `username`
```

Comment:
We should not use defaults where they're not helping users e.g. username, password etc.
I created https://github.com/elastic/connectors-python/issues/606 for adjusting existing connectors, let's make sure new connectors will align to those rules, and we document examples where necessary

Reply:
As of now default values are used to run e2e and the PR for similar change in Jira and SharePoint is still draft https://github.com/elastic/connectors-python/pull/648, shall we wait for this to get merged ?

Reply:
[6d316f7](https://github.com/elastic/connectors-python/pull/647/commits/6d316f71c195ad4a7d3ccb23819882b9c9e462e3)